qemu/tests/cfg/numa_opts.cfg

```diff
@@ -13,20 +11,30 @@
   use_mem = no
   variants:
     - nodes.0:
-      # no extra parameters => zero nodes
-      numa_nodes = 0
-      # on ppc,no extra parameters => one node with size = mem
+      # no extra parameters => one node in guest os
+      guest_expect_nodes = 1
+      # no extra parameters => zero nodes with qemu monitor
+      monitor_expect_nodes = 0
+      # on ppc,no extra parameters => one node and size = mem with qemu monitor
       ppc64le, ppc64:
-          numa_nodes = 1
+          monitor_expect_nodes = 1
+      variants:
+          - @default:
+          - with_maxmem:
+              required_qemu = [5.1.0,)
+              # no extra parameters, with maxmem => one node in monitor, guest
+              maxmem_mem = 32G
+              slots_mem = 4
```

Comment:
We don't need slots option for this case.

Reply:
> We don't need slots option for this case.

It is must in automation, otherwise, maxmem can not be added:
https://github.com/avocado-framework/avocado-vt/blob/e4e7aad472a0e5790832e0fe65ad98e63a7ec1dc/virttest/qemu_vm.py#L961

I do not know the reason of the design above, but with `slots` in this case do not have side effect as my understanding.

Reply:
The design should not be right. This case need test without slots(explicitly expressed in qemu commit).

Reply:
> The design should not be right. This case need test without slots(explicitly expressed in qemu commit).

Ok, that is another issue, which is not related with this test scenario, i can remove the `slots_mem` in this case,  means it will be blocked by the new issue now.
could you create a new bz and give the detailed expect result ? Thanks!

Reply:
@huangyum Updated, remove the `slots`, could you update the test case to highlight this test scenario must without slots, since `explicitly expressed in qemu commit`, thanks!

Now, test case `numa_opts.nodes.0.with_maxmem` will failed: maxmem can not be added in qemu cli, expect node is 1 while the actual node in monitor is 0 for x86.

Reply:
Sure, reported bug 1869589.

Reply:
For the new issue 1869589, patch merged: https://github.com/avocado-framework/avocado-vt/pull/2718